components/usage/pkg/apiv1/usage_test.go

```diff
@@ -204,112 +201,87 @@ func TestUsageService_ListBilledUsage_Pagination(t *testing.T) {
 	}
 
 	scenarios := []Scenario{
-		(func() Scenario {
```

Comment:
This wrapping in a self-invoking anonymous function isn't needed. It would be useful we wanted to do some imperative work, but we're doing only declarative statements.

Reply:
💡  thanks 💯